### PR TITLE
Merging Spotify part of sentiment library  and Spotify function 

### PR DESCRIPTION
**Pull request by matildeyang**

### CODE REVIEW

File: js/functions/spotifyColors.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+window.functions.spotifyColors = {
+  name: 'spotify colors',
+  type: 'Edit',
+  run: async function () {
+    const token = window.localStorage.getItem('spotifyToken')
+    if (!token) {
+      const answer = window.confirm('Do you want to authenticate Spotify?')
+      console.log(answer)
+      if (answer) {
+        window.Sent.spotifyAuth('d41b83c2f2224356bd94fbf83924e6d4', 'https://artware.app')
```
Comment (nbriz) [inline js/functions/spotifyColors.js]: think this should be using the new `spotifyMoodColors` library right? not the old `Sent` library,
```js
window.SpotifyMoodColors.spotifyAuth('d41b83c2f2224356bd94fbf83924e6d4', 'https://artware.app')
```

File: js/functions/spotifyColors.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      }
+    } else {
+      const colors = await window.Sent.getColors()
```
Comment (nbriz) [inline js/functions/spotifyColors.js]: same here, should be...
```js
const colors = await window.SpotifyMoodColors.getColors()
```

**Review by nbriz — CHANGES_REQUESTED** [review]
hey Mati, think this is very close! if u can make the following changes (sooner rather than later so that i can merge ethan + leandra's changes) that be great!

first, delete ur `js/libs/sentiment.js` file, b/c that's going to create a conflict w/ethan + leandra's file. (might also be best to remove the old docs file as well `docs/Sentiment.md` to be safe)

then, after u delete the old `sentiment.js` file, u'll need to update the `index.html` file [here](https://github.com/matildeyang/paintArtware1.0/blob/3e8ebc06d0aa82d524e91c6e44e7c75e5802adac/index.html#L50), so that u're loading ur library:
```HTML
<script src="/js/libs/spotifyMoodColors.js"></script>
```
instead of the old `sentiment.js` file
```HTML
<script src="/js/libs/sentiment.js"></script>
```

when u do that, u'll also need to make sure ur `js/functions/spotifyColors.js` logic is using ur new `SpotifyMoodColors` library && not the old `sentiment.js` library (see my other comments below)

File: js/functions/spotifyColors.js (hunk 2 above)
Comment (matildeyang) [reply js/functions/spotifyColors.js]: I modified everything last night, let me know if other changes are needed!

**Comment by nbriz** [comment]
great! merging now

**Pull request merged**